quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java

```diff
@@ -32,6 +35,9 @@
     @Autowired
     SignupBusinessService signupBusinessService;
 
+    @Autowired
+    SignOutBusinessService signOutBusinessService;
```

Comment:
Where is signout business service file ?

Reply:
I have added now.
Kindly check